assets/queries/ansible/aws/sqs_policy_with_public_access/query.rego

```diff
@@ -0,0 +1,45 @@
+package Cx
+
+
+CxPolicy [ result ] {
+  document := input.document[i]
+  tasks := getTasks(document)
+  task := tasks[t]
+  sqsPolicy := task["community.aws.sqs_queue"]
+  sqsPolicyName := task.name
+  contains(sqsPolicy.policy.Statement[_].Principal,"*")
+
+
+	result := {
+                "documentId": 		  input.document[i].id,
+                "searchKey": 	      sprintf("name=%s.{{community.aws.sqs_queue}}.policy.Principal", [sqsPolicyName]),
+                "issueType":		    "IncorrectValue",
+                "keyExpectedValue": sprintf("name=%s.{{community.aws.sqs_queue}}.policy.Principal should not be equal to '*'", [sqsPolicyName]),
+                "keyActualValue": 	sprintf("name=%s.{{community.aws.sqs_queue}}.policy.Principal is equal to '*'", [sqsPolicyName]),
+              }
+}
+CxPolicy [ result] {
+  document := input.document[i]
+  tasks := getTasks(document)
+  task := tasks[t]
+  sqsPolicy := task["community.aws.sqs_queue"]
+  sqsPolicyName := task.name
+  contains(sqsPolicy.policy.Statement[_].Principal[j].AWS,"*")
```

Comment:
Principal[j].AWS can also be an array we should check for Principal[j].AWS[_] as well